airflow/hooks/presto_hook.py

```diff
@@ -44,18 +45,22 @@ class PrestoHook(DbApiHook):
     def get_conn(self):
         """Returns a connection object"""
         db = self.get_connection(self.presto_conn_id)
-        reqkwargs = None
+        auth = None
         if db.password is not None:
-            reqkwargs = {'auth': HTTPBasicAuth(db.login, db.password)}
-        return presto.connect(
+            auth = prestodb.auth.BasicAuthentication(db.login, db.password)
+
+        isolation_level = db.extra_dejson.get('isolation_level', 'AUTOCOMMIT').upper()
+        return prestodb.dbapi.connect(
             host=db.host,
             port=db.port,
-            username=db.login,
+            user=db.login,
             source=db.extra_dejson.get('source', 'airflow'),
-            protocol=db.extra_dejson.get('protocol', 'http'),
+            http_scheme=db.extra_dejson.get('protocol', 'http'),
             catalog=db.extra_dejson.get('catalog', 'hive'),
-            requests_kwargs=reqkwargs,
-            schema=db.schema)
+            schema=db.schema,
+            auth=auth,
+            isolation_level=getattr(IsolationLevel, isolation_level, 0)
```

Comment:
I would suggest `isolation_level.IsolationLevel or 0` instead. IMO `getattr` often makes code a bit harder to read.

Reply:
I think `IsolationLevel. isolation_level` won't work because isolation_level is user input and it's in string format.

Reply:
Oh, I see. 👍